### ISSUE TITLE
Unable to test with jest due to parent having no testID

### ISSUE THREAD

**Issue by mike-skwi** [DylanVann/react-native-fast-image#810]
Hello. I'm trying to do some testing with jest/testing library. Usually, I'd try something with `findByTestId`, but passing a testID to <FastImage /> only gives it to the child and not the parent view. The issue here is that passing a style into FastImage passes it to the parent above it, which I am unable to access and thus preventing me from testing. See below, this is my debug() from testing-library. 

`<View
        style={
          Array [
            Object {
              "overflow": "hidden",
            },
            Array [
              Object {
                "borderRadius": 46,
                "height": 92,
                "width": 92,
              },
              undefined,
            ],
          ]
        }
      >
        <FastImageView
          resizeMode="cover"
          source={null}
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
          testID="Avatar"
        />
      </View>`
      

     TL:DR: Would it be possible to simply add a testID to the parent of FastImageView? If there's a different way of testing this I should be aware of, please let me know. I've tried using container etc but nothing has worked.

**Comment by vasylnahuliak** [comment]
`jest.mock('react-native-fast-image', () => 'FastImage');`

Try add mock
